pyproject.toml

```diff
@@ -58,7 +58,8 @@ include = [
   "pyccel/stdlib/**/*.c",
   "pyccel/stdlib/**/*.f90",
   "pyccel/extensions/STC/include",
-  "pyccel/extensions/gFTL/include/v2"
+  "pyccel/extensions/gFTL/include/v2",
+  "pyccel/stdlib/cuda_ndarrays/cuda_ndarrays.cu"
```

Comment:
Please group this with the other stdlib files. I think we can safely include all cuda files found inside stdlib.